trace-normalization/src/normalizer.rs

```diff
@@ -16,8 +16,18 @@ const YEAR_2000_NANOSEC_TS: i64 = 946684800000000000;
 // DEFAULT_SPAN_NAME is the default name we assign a span if it's missing and we have no reasonable fallback
 const DEFAULT_SPAN_NAME: &str = "unnamed_operation";
 
+const TAG_SAMPLING_PRIORITY: &str = "_sampling_priority_v1";
+const TAG_ORIGIN: &str = "_dd.origin";
+
+#[allow(dead_code)]
```

Comment:
maybe an enum would make sense? (and will avoid the allow dead code)

Reply:
totally, I'll change it